# Add searx
# This will become the default engine if no other engine is already set

--- a/components/search_engines/prepopulated_engines.json
+++ b/components/search_engines/prepopulated_engines.json
@@ -34,6 +34,16 @@
   // The following engines are included in country lists and are added to the
   // list of search engines on the first run depending on user's country.
   "elements": {
+    "searx": {
+      "name": "Searx",
+      "keyword": "searx.me",
+      "favicon_url": "https://searx.me/favicon.ico",
+      "search_url": "https://searx.me/?q={searchTerms}",
+      "new_tab_url": "https://searx.me/",
+      "type": "SEARCH_ENGINE_SEARX",
+      "id": 12
+    },
+
     "ask": {
       "name": "Ask",
       "keyword": "ask.com",
--- a/components/search_engines/search_engine_type.h
+++ b/components/search_engines/search_engine_type.h
@@ -47,6 +47,7 @@ enum SearchEngineType {
   SEARCH_ENGINE_SAPO,
   SEARCH_ENGINE_SEARCHNU,
   SEARCH_ENGINE_SEARCH_RESULTS,
+  SEARCH_ENGINE_SEARX,
   SEARCH_ENGINE_SEZNAM,
   SEARCH_ENGINE_SNAPDO,
   SEARCH_ENGINE_SOFTONIC,
--- a/components/search_engines/template_url_prepopulate_data.cc
+++ b/components/search_engines/template_url_prepopulate_data.cc
@@ -29,6 +29,7 @@ namespace {
 
 // Default (for countries with no better engine set)
 const PrepopulatedEngine* const engines_default[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -39,6 +40,7 @@ const PrepopulatedEngine* const engines_
 // Engine selection by country ------------------------------------------------
 // United Arab Emirates
 const PrepopulatedEngine* const engines_AE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -48,6 +50,7 @@ const PrepopulatedEngine* const engines_
 
 // Albania
 const PrepopulatedEngine* const engines_AL[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -57,6 +60,7 @@ const PrepopulatedEngine* const engines_
 
 // Argentina
 const PrepopulatedEngine* const engines_AR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -66,6 +70,7 @@ const PrepopulatedEngine* const engines_
 
 // Austria
 const PrepopulatedEngine* const engines_AT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -75,6 +80,7 @@ const PrepopulatedEngine* const engines_
 
 // Australia
 const PrepopulatedEngine* const engines_AU[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -84,6 +90,7 @@ const PrepopulatedEngine* const engines_
 
 // Bosnia and Herzegovina
 const PrepopulatedEngine* const engines_BA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -93,6 +100,7 @@ const PrepopulatedEngine* const engines_
 
 // Belgium
 const PrepopulatedEngine* const engines_BE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -102,6 +110,7 @@ const PrepopulatedEngine* const engines_
 
 // Bulgaria
 const PrepopulatedEngine* const engines_BG[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -111,6 +120,7 @@ const PrepopulatedEngine* const engines_
 
 // Bahrain
 const PrepopulatedEngine* const engines_BH[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -121,6 +131,7 @@ const PrepopulatedEngine* const engines_
 
 // Burundi
 const PrepopulatedEngine* const engines_BI[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -131,6 +142,7 @@ const PrepopulatedEngine* const engines_
 
 // Brunei
 const PrepopulatedEngine* const engines_BN[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -140,6 +152,7 @@ const PrepopulatedEngine* const engines_
 
 // Bolivia
 const PrepopulatedEngine* const engines_BO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -149,6 +162,7 @@ const PrepopulatedEngine* const engines_
 
 // Brazil
 const PrepopulatedEngine* const engines_BR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -158,6 +172,7 @@ const PrepopulatedEngine* const engines_
 
 // Belarus
 const PrepopulatedEngine* const engines_BY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_by,
@@ -168,6 +183,7 @@ const PrepopulatedEngine* const engines_
 
 // Belize
 const PrepopulatedEngine* const engines_BZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -177,6 +193,7 @@ const PrepopulatedEngine* const engines_
 
 // Canada
 const PrepopulatedEngine* const engines_CA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -186,6 +203,7 @@ const PrepopulatedEngine* const engines_
 
 // Switzerland
 const PrepopulatedEngine* const engines_CH[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -195,6 +213,7 @@ const PrepopulatedEngine* const engines_
 
 // Chile
 const PrepopulatedEngine* const engines_CL[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -204,6 +223,7 @@ const PrepopulatedEngine* const engines_
 
 // China
 const PrepopulatedEngine* const engines_CN[] = {
+    &searx,
     &duckduckgo,
     &baidu,
     &sogou,
@@ -214,6 +234,7 @@ const PrepopulatedEngine* const engines_
 
 // Colombia
 const PrepopulatedEngine* const engines_CO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -223,6 +244,7 @@ const PrepopulatedEngine* const engines_
 
 // Costa Rica
 const PrepopulatedEngine* const engines_CR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -232,6 +254,7 @@ const PrepopulatedEngine* const engines_
 
 // Czech Republic
 const PrepopulatedEngine* const engines_CZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &seznam_cz,
@@ -241,6 +264,7 @@ const PrepopulatedEngine* const engines_
 
 // Germany
 const PrepopulatedEngine* const engines_DE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -250,6 +274,7 @@ const PrepopulatedEngine* const engines_
 
 // Denmark
 const PrepopulatedEngine* const engines_DK[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -259,6 +284,7 @@ const PrepopulatedEngine* const engines_
 
 // Dominican Republic
 const PrepopulatedEngine* const engines_DO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -268,6 +294,7 @@ const PrepopulatedEngine* const engines_
 
 // Algeria
 const PrepopulatedEngine* const engines_DZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -277,6 +304,7 @@ const PrepopulatedEngine* const engines_
 
 // Ecuador
 const PrepopulatedEngine* const engines_EC[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -286,6 +314,7 @@ const PrepopulatedEngine* const engines_
 
 // Estonia
 const PrepopulatedEngine* const engines_EE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -296,6 +325,7 @@ const PrepopulatedEngine* const engines_
 
 // Egypt
 const PrepopulatedEngine* const engines_EG[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -305,6 +335,7 @@ const PrepopulatedEngine* const engines_
 
 // Spain
 const PrepopulatedEngine* const engines_ES[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -314,6 +345,7 @@ const PrepopulatedEngine* const engines_
 
 // Finland
 const PrepopulatedEngine* const engines_FI[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -323,6 +355,7 @@ const PrepopulatedEngine* const engines_
 
 // Faroe Islands
 const PrepopulatedEngine* const engines_FO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -332,6 +365,7 @@ const PrepopulatedEngine* const engines_
 
 // France
 const PrepopulatedEngine* const engines_FR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -342,6 +376,7 @@ const PrepopulatedEngine* const engines_
 
 // United Kingdom
 const PrepopulatedEngine* const engines_GB[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -351,6 +386,7 @@ const PrepopulatedEngine* const engines_
 
 // Greece
 const PrepopulatedEngine* const engines_GR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -360,6 +396,7 @@ const PrepopulatedEngine* const engines_
 
 // Guatemala
 const PrepopulatedEngine* const engines_GT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -369,6 +406,7 @@ const PrepopulatedEngine* const engines_
 
 // Hong Kong
 const PrepopulatedEngine* const engines_HK[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_hk,
@@ -379,6 +417,7 @@ const PrepopulatedEngine* const engines_
 
 // Honduras
 const PrepopulatedEngine* const engines_HN[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -388,6 +427,7 @@ const PrepopulatedEngine* const engines_
 
 // Croatia
 const PrepopulatedEngine* const engines_HR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -397,6 +437,7 @@ const PrepopulatedEngine* const engines_
 
 // Hungary
 const PrepopulatedEngine* const engines_HU[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -406,6 +447,7 @@ const PrepopulatedEngine* const engines_
 
 // Indonesia
 const PrepopulatedEngine* const engines_ID[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_id,
@@ -415,6 +457,7 @@ const PrepopulatedEngine* const engines_
 
 // Ireland
 const PrepopulatedEngine* const engines_IE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -424,6 +467,7 @@ const PrepopulatedEngine* const engines_
 
 // Israel
 const PrepopulatedEngine* const engines_IL[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -433,6 +477,7 @@ const PrepopulatedEngine* const engines_
 
 // India
 const PrepopulatedEngine* const engines_IN[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_in,
@@ -442,6 +487,7 @@ const PrepopulatedEngine* const engines_
 
 // Iraq
 const PrepopulatedEngine* const engines_IQ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -451,6 +497,7 @@ const PrepopulatedEngine* const engines_
 
 // Iran
 const PrepopulatedEngine* const engines_IR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -461,6 +508,7 @@ const PrepopulatedEngine* const engines_
 
 // Iceland
 const PrepopulatedEngine* const engines_IS[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -470,6 +518,7 @@ const PrepopulatedEngine* const engines_
 
 // Italy
 const PrepopulatedEngine* const engines_IT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -479,6 +528,7 @@ const PrepopulatedEngine* const engines_
 
 // Jamaica
 const PrepopulatedEngine* const engines_JM[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -488,6 +538,7 @@ const PrepopulatedEngine* const engines_
 
 // Jordan
 const PrepopulatedEngine* const engines_JO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -497,6 +548,7 @@ const PrepopulatedEngine* const engines_
 
 // Japan
 const PrepopulatedEngine* const engines_JP[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_jp,
@@ -507,6 +559,7 @@ const PrepopulatedEngine* const engines_
 
 // Kenya
 const PrepopulatedEngine* const engines_KE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -516,6 +569,7 @@ const PrepopulatedEngine* const engines_
 
 // South Korea
 const PrepopulatedEngine* const engines_KR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &naver,
@@ -526,6 +580,7 @@ const PrepopulatedEngine* const engines_
 
 // Kuwait
 const PrepopulatedEngine* const engines_KW[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -535,6 +590,7 @@ const PrepopulatedEngine* const engines_
 
 // Kazakhstan
 const PrepopulatedEngine* const engines_KZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_kz,
@@ -545,6 +601,7 @@ const PrepopulatedEngine* const engines_
 
 // Lebanon
 const PrepopulatedEngine* const engines_LB[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -554,6 +611,7 @@ const PrepopulatedEngine* const engines_
 
 // Liechtenstein
 const PrepopulatedEngine* const engines_LI[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -563,6 +621,7 @@ const PrepopulatedEngine* const engines_
 
 // Lithuania
 const PrepopulatedEngine* const engines_LT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -572,6 +631,7 @@ const PrepopulatedEngine* const engines_
 
 // Luxembourg
 const PrepopulatedEngine* const engines_LU[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -581,6 +641,7 @@ const PrepopulatedEngine* const engines_
 
 // Latvia
 const PrepopulatedEngine* const engines_LV[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_ru,
@@ -590,6 +651,7 @@ const PrepopulatedEngine* const engines_
 
 // Libya
 const PrepopulatedEngine* const engines_LY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -599,6 +661,7 @@ const PrepopulatedEngine* const engines_
 
 // Morocco
 const PrepopulatedEngine* const engines_MA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -608,6 +671,7 @@ const PrepopulatedEngine* const engines_
 
 // Monaco
 const PrepopulatedEngine* const engines_MC[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -617,6 +681,7 @@ const PrepopulatedEngine* const engines_
 
 // Moldova
 const PrepopulatedEngine* const engines_MD[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_ru,
@@ -626,6 +691,7 @@ const PrepopulatedEngine* const engines_
 
 // Montenegro
 const PrepopulatedEngine* const engines_ME[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -635,6 +701,7 @@ const PrepopulatedEngine* const engines_
 
 // Macedonia
 const PrepopulatedEngine* const engines_MK[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -644,6 +711,7 @@ const PrepopulatedEngine* const engines_
 
 // Mexico
 const PrepopulatedEngine* const engines_MX[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -653,6 +721,7 @@ const PrepopulatedEngine* const engines_
 
 // Malaysia
 const PrepopulatedEngine* const engines_MY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -662,6 +731,7 @@ const PrepopulatedEngine* const engines_
 
 // Nicaragua
 const PrepopulatedEngine* const engines_NI[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -671,6 +741,7 @@ const PrepopulatedEngine* const engines_
 
 // Netherlands
 const PrepopulatedEngine* const engines_NL[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -680,6 +751,7 @@ const PrepopulatedEngine* const engines_
 
 // Norway
 const PrepopulatedEngine* const engines_NO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -689,6 +761,7 @@ const PrepopulatedEngine* const engines_
 
 // New Zealand
 const PrepopulatedEngine* const engines_NZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -698,6 +771,7 @@ const PrepopulatedEngine* const engines_
 
 // Oman
 const PrepopulatedEngine* const engines_OM[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -707,6 +781,7 @@ const PrepopulatedEngine* const engines_
 
 // Panama
 const PrepopulatedEngine* const engines_PA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -716,6 +791,7 @@ const PrepopulatedEngine* const engines_
 
 // Peru
 const PrepopulatedEngine* const engines_PE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -725,6 +801,7 @@ const PrepopulatedEngine* const engines_
 
 // Philippines
 const PrepopulatedEngine* const engines_PH[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_ph,
@@ -734,6 +811,7 @@ const PrepopulatedEngine* const engines_
 
 // Pakistan
 const PrepopulatedEngine* const engines_PK[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -743,6 +821,7 @@ const PrepopulatedEngine* const engines_
 
 // Poland
 const PrepopulatedEngine* const engines_PL[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -752,6 +831,7 @@ const PrepopulatedEngine* const engines_
 
 // Puerto Rico
 const PrepopulatedEngine* const engines_PR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -761,6 +841,7 @@ const PrepopulatedEngine* const engines_
 
 // Portugal
 const PrepopulatedEngine* const engines_PT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -770,6 +851,7 @@ const PrepopulatedEngine* const engines_
 
 // Paraguay
 const PrepopulatedEngine* const engines_PY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -779,6 +861,7 @@ const PrepopulatedEngine* const engines_
 
 // Qatar
 const PrepopulatedEngine* const engines_QA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -788,6 +871,7 @@ const PrepopulatedEngine* const engines_
 
 // Romania
 const PrepopulatedEngine* const engines_RO[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -797,6 +881,7 @@ const PrepopulatedEngine* const engines_
 
 // Serbia
 const PrepopulatedEngine* const engines_RS[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -806,6 +891,7 @@ const PrepopulatedEngine* const engines_
 
 // Russia
 const PrepopulatedEngine* const engines_RU[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_ru,
@@ -816,6 +902,7 @@ const PrepopulatedEngine* const engines_
 
 // Rwanda
 const PrepopulatedEngine* const engines_RW[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -825,6 +912,7 @@ const PrepopulatedEngine* const engines_
 
 // Saudi Arabia
 const PrepopulatedEngine* const engines_SA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -834,6 +922,7 @@ const PrepopulatedEngine* const engines_
 
 // Sweden
 const PrepopulatedEngine* const engines_SE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -843,6 +932,7 @@ const PrepopulatedEngine* const engines_
 
 // Singapore
 const PrepopulatedEngine* const engines_SG[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -853,6 +943,7 @@ const PrepopulatedEngine* const engines_
 
 // Slovenia
 const PrepopulatedEngine* const engines_SI[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -862,6 +953,7 @@ const PrepopulatedEngine* const engines_
 
 // Slovakia
 const PrepopulatedEngine* const engines_SK[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -871,6 +963,7 @@ const PrepopulatedEngine* const engines_
 
 // El Salvador
 const PrepopulatedEngine* const engines_SV[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -880,6 +973,7 @@ const PrepopulatedEngine* const engines_
 
 // Syria
 const PrepopulatedEngine* const engines_SY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -889,6 +983,7 @@ const PrepopulatedEngine* const engines_
 
 // Thailand
 const PrepopulatedEngine* const engines_TH[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_th,
@@ -898,6 +993,7 @@ const PrepopulatedEngine* const engines_
 
 // Tunisia
 const PrepopulatedEngine* const engines_TN[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo,
@@ -907,6 +1003,7 @@ const PrepopulatedEngine* const engines_
 
 // Turkey
 const PrepopulatedEngine* const engines_TR[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_tr,
@@ -916,6 +1013,7 @@ const PrepopulatedEngine* const engines_
 
 // Trinidad and Tobago
 const PrepopulatedEngine* const engines_TT[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -925,6 +1023,7 @@ const PrepopulatedEngine* const engines_
 
 // Taiwan
 const PrepopulatedEngine* const engines_TW[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_tw,
@@ -935,6 +1034,7 @@ const PrepopulatedEngine* const engines_
 
 // Tanzania
 const PrepopulatedEngine* const engines_TZ[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -944,6 +1044,7 @@ const PrepopulatedEngine* const engines_
 
 // Ukraine
 const PrepopulatedEngine* const engines_UA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yandex_ua,
@@ -954,6 +1055,7 @@ const PrepopulatedEngine* const engines_
 
 // United States
 const PrepopulatedEngine* const engines_US[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -963,6 +1065,7 @@ const PrepopulatedEngine* const engines_
 
 // Uruguay
 const PrepopulatedEngine* const engines_UY[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -972,6 +1075,7 @@ const PrepopulatedEngine* const engines_
 
 // Venezuela
 const PrepopulatedEngine* const engines_VE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &yahoo_ve,
@@ -981,6 +1085,7 @@ const PrepopulatedEngine* const engines_
 
 // Vietnam
 const PrepopulatedEngine* const engines_VN[] = {
+    &searx,
     &duckduckgo,
     &google,
     &coccoc,
@@ -991,6 +1096,7 @@ const PrepopulatedEngine* const engines_
 
 // Yemen
 const PrepopulatedEngine* const engines_YE[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -1000,6 +1106,7 @@ const PrepopulatedEngine* const engines_
 
 // South Africa
 const PrepopulatedEngine* const engines_ZA[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
@@ -1009,6 +1116,7 @@ const PrepopulatedEngine* const engines_
 
 // Zimbabwe
 const PrepopulatedEngine* const engines_ZW[] = {
+    &searx,
     &duckduckgo,
     &google,
     &bing,
